com.unity.render-pipelines.high-definition/Editor/PostProcessing/PaniniProjectionEditor.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEditor.Rendering;
+using UnityEngine;
+using UnityEngine.Rendering.HighDefinition;
+
+namespace UnityEditor.Rendering.HighDefinition
+{
+    [VolumeComponentEditor(typeof(PaniniProjection))]
```

Comment:
If you merge my PR first, you will need to add the VolumeComponentMenu attribute and specify that this Volume Component is only supported on HDRP, If not, users using URP will be able to add it.